README.md

```diff
@@ -21,14 +21,18 @@ A tool for quantum error correcting codes and numerical simulations developed by
 [Chair for Design Automation](https://www.cda.cit.tum.de/) at the [Technical University of Munich](https://www.tum.de/)
 based on methods proposed in [[1]](https://arxiv.org/abs/2209.01180). QECC is part of the Munich Quantum Toolkit (MQT).
 
-The tool can be used to decode quantum LDPC codes and conduct respective numerical simulations.
-
-At the moment the general QLDPC
-decoder [[2]](https://ieeexplore.ieee.org/abstract/document/9682738)
-and a heuristic (which improves the runtime of the algorithm) [[1]](https://arxiv.org/abs/2209.01180) are implemented.
-Currently, open-source software by Joschka Roffe et
-al.: [[3]](https://github.com/quantumgizmos/bias_tailored_qldpc) is used to construct codes (toric, lifted product and
-hypergraph product).
+The tool can be used to:
+
+- Decode quantum LDPC codes and conduct respective numerical simulations.
+  - At the moment the general QLDPC
+    decoder [[2]](https://ieeexplore.ieee.org/abstract/document/9682738)
+    and a heuristic (which improves the runtime of the algorithm) [[1]](https://arxiv.org/abs/2209.01180) are
+    implemented.
+    Currently, open-source software by Joschka Roffe et
+    al.: [[3]](https://github.com/quantumgizmos/bias_tailored_qldpc) is used to construct codes (toric, lifted product
+    and
+    hypergraph product).
+- Apply error correction to quantum circuits.
```

Comment:
This could probably use a little more documentation and a reference to the corresponding paper ("... based on [...]").